src/extension/background-script/actions/accounts/add.ts

```diff
@@ -11,9 +13,11 @@ const add = async (message, sender) => {
   const password = state.getState().password;
   const currentAccountId = state.getState().currentAccountId;
 
-  const accountId = Object.keys(accounts).length + 1;
-  console.log(`Created account ${accountId}`);
+  if (!password) throw new Error(
```

Comment:
I think we need to return something like `{ error: "Password is missing" }`; 
Because of the messaging required from the "frontend" to the backrgound script we can not throw an error I think. (this might be handled on a different level, but I am not sure)

Reply:
Updated it.

Looking at i.e. this, both is being used:  
- https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/router.js#L60
- https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/router.js#L71

Some other places in `background-script` is using both as well.  
@dylancom do you know when which is being used or should be used or how it's propagated in a way?

Reply:
Can be discussed later on